examples/terraform/README.md

```diff
@@ -0,0 +1,67 @@
+# How to Deploy Talos Cluster in Public Cloud Platforms
+
+In this guide, we will learn how to deploy Talos resources by using Terraform with cloud platforms.
+
+You can look at [terraform module docs](https://registry.terraform.io/providers/siderolabs/talos/latest/docs) which provide explanations on guides, resources and data sources as the latest version.
+
+### How to Deploy Talos Cluster in AWS
+
+#### Prerequisites
+
+- AWS Account 
+- AWS CLI
+- Getting keys from AWS IAM and configuring it locally
+- Terraform
+- kubectl
+- talosctl
+- SSH key to connect to EC2 instances in talos (if necessary, use secrets manager to secure your secrets)
+
+#### Setting up the environment
+
+- Clone the [Sidero Labs' contribution repo](https://github.com/siderolabs/contrib) to your GitHub Account and then clone the repo on your computer.
+- Then, write ```cd contrib``` to go to the directory
```

Comment:
I wouldn't say "write" in each step. Instead I would put the commands needed in a code block so people can copy and paste them easily.

After you clone the repository locally you can move into the aws folder and initialize terraform
```bash
cd contrib/examples/terraform/aws
terraform init
```
Make any changes you need to add networks and configuration you need and then apply the resources and deploy the Cloud Controller Manager (CCM)
```bash
terraform apply
kubectl apply -f manifests/ccm.yaml
```

I think you could just do that once and say it applies to each cloud provider without having to repeat yourself.